docker/scripts/cloud-init.cfg

```diff
@@ -40,9 +40,9 @@ write_files:
       ExecStartPre=/usr/bin/docker-credential-gcr configure-docker --registries us-central1-docker.pkg.dev
       ExecStartPre=mount --bind /var/lib/nvidia /var/lib/nvidia
       ExecStartPre=mount -o remount,exec /var/lib/nvidia
-      ExecStartPre=/usr/bin/docker pull us-central1-docker.pkg.dev/training-algorithms-external/mlcommons-docker-repo/mlcommons:ogbg
+      ExecStartPre=/usr/bin/docker pull us-central1-docker.pkg.dev/training-algorithms-external/mlcommons-docker-repo/mlcommons:criteo2
```

Comment:
why criteo2 and not criteo?

Reply:
criteo was the other image I was trying (locally build dataset copy to docker image, push to artifact registry etc) that path didn't work 

will update docker tags and update this script